tests/test_callprofiler.py

```diff
@@ -380,4 +380,47 @@ def foo():
         modmesh.call_profiler.reset()
         foo()
 
+    def test_status(self):
```

Comment:
Please make inline annotation for what is changed in the test code.  You moved the code and there is no diff.